db/migrate/20241009151640_create_discretionary_disregards.rb

```diff
@@ -0,0 +1,18 @@
+class CreateDiscretionaryDisregards < ActiveRecord::Migration[7.2]
+  def change
+    create_table :discretionary_disregards, id: :uuid do |t|
```

Comment:
as mentioned below disrectionary_policy_disregards would separate this naming from the notion of "disregarded benefits"